packages/discord-argument/src/lib/structures/Command.ts

```diff
@@ -0,0 +1,66 @@
+import type { Message } from 'discord.js';
```

Comment:
Idk about this tbh. Pretty much what Vladdy said. It feels wrong to have that here. Technically not an issue to framework because it can extend and then export but ... I just don't know. I want to hear @vladfrangu  and @kyranet's opinions here

Reply:
I don't like this at all. If we really want to abstract args out of the framework, it should provide a way to register resolvers by name (`argsResolvers.register('name', IArgument)`) and use that instead. In no way are we re-extending in framework from a module